internal/factory/resource_names.go

```diff
@@ -0,0 +1,37 @@
+package factory
+
+import (
+	"fmt"
+
+	"github.com/openshift/cluster-logging-operator/internal/constants"
+)
+
+func GenerateResourceNames(name, namespace string) map[string]string {
```

Comment:
Minor: Would be more type safe to return a  struct with string fields rather than a map with const string keys.

Reply:
This is a good suggestion!